gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-oidc/src/test/java/io/gravitee/am/gateway/handler/oauth2/service/granter/uma/UmaTokenGranterTest.java

```diff
@@ -266,6 +280,7 @@ public void grant_rptNotBelongToSameUser() {
 
     @Test
     public void grant_user_nominalCase() {
+        when(resourceService.findAccessPoliciesByResources(anyList())).thenReturn(Single.just(Collections.emptyList()));
```

Comment:
Please remove it from the test method to the setup part

Reply:
FP